operator/controllers/ambassador.go

```diff
@@ -115,6 +125,35 @@ func getAmbassadorCircuitBreakerConfig(
 	return nil, nil
 }
 
+// Return TLSContext configuration if SSL is enabled with Ambassador and returns empty string if SSL is not enabled
+func getAmbassadorTLSContextConfig(mlDep *machinelearningv1.SeldonDeployment, p *machinelearningv1.PredictorSpec, addNamespace bool) (string, error) {
+
+	if p.SSL == nil || len(p.SSL.CertSecretName) <= 0 {
+		return "", nil
+	}
+
+	name := p.Name
+	namespace := getNamespace(mlDep)
+
+	c := AmbassadorTLSContextConfig{
+		ApiVersion: "ambassador/v1",
+		Kind:       "TLSContext",
+		Name:       "seldon_" + namespace + "_" + mlDep.ObjectMeta.Name + "_" + name + "_tls_config",
+		Hosts:      []string{},
+		Secret:     p.SSL.CertSecretName,
+	}
+
+	if addNamespace {
+		c.Name = "seldon_" + namespace + "_" + mlDep.ObjectMeta.Name + "_" + name + "_tls_config"
+	}
```

Comment:
It seems to be doing the same thing above. Should we get rid of this one?

Reply:
Good catch!